build/root/BUILD.root

```diff
@@ -66,6 +66,11 @@ filegroup(
         "//cmd:all-srcs",
         "//docs:all-srcs",
         "//hack:all-srcs",
+        "//kubernetes/cluster:all-srcs",
+        "//kubernetes/docs:all-srcs",
+        "//kubernetes/examples:all-srcs",
+        "//kubernetes/hack/lib:all-srcs",
+        "//kubernetes/third_party/htpasswd:all-srcs",
```

Comment:
hummm, no sure if these are needed..Seems irrelevant to what is changed in this PR.